content/blog/hpe-oneview-and-servicenow-integration-with-stackstorm.md

```diff
@@ -0,0 +1,46 @@
+---
+title: HPE OneView and ServiceNow integration with Stackstorm
+date: 2021-09-29T17:53:58.637Z
+author: Rick Kauffman
+authorimage: /img/Avatar2.svg
+---
+![](http://www.techworldwookie.com/blogpost/flowchart.png)
+
+
+HPE OneView is a powerful infrastructure automation/management platform used with HPE DL servers and HPE Synergy products. I started thinking, how would I get all the alarms and 'automatically' save them as records in a ServiceNow table. Naturally, my second thought was how can I tweet some vlan information into OneView! 
+
+HPE Oneview has a powerful Restful API and it can be used to get the information in and out of OneView. ServiceNow has a powerful Restful API as well. All that is needed is some middleware and a couple of python bindings (python code that abstracts the API). Turns out the python bindings are already written for both! Seems easy enough, write a couple of python scripts, and we should be good to go, or GTG if you're hip and cool.
+
+This would involve writing the code for **both systems**. What if we leveraged Stackstorm? A quick check of the Stackstorm exchange [exchange.stackstorm.com](exchange.stackstorm.com) and we find that there is Stackstorm integration pack available for ServiceNow. If we use Stackstorm, we only have to write **half** the code. All that needs done is to write the code for a HPE OneView Stackstorm integration pack. 
+
+I have written a couple of other blogs on Stackstorm and if you are interested in trying this approach, I suggest you go to <https://developer.hpe.com/blog/tag/stackstorm> and read my other posts.
+
+Developing the **stackstorm-hpe-oneview** integration pack, which is available here: <https://github.com/HewlettPackard/stackstorm-hpe-oneview>, is fairly straightforward. For this integration to function we will have to write five very short actions and a couple of simple rules. You can see in the chart at the top that two of the actions will be used with the first workflow and three will be need for the second workflow. Actions are the workhorse of Stackstorm. They are basically the same thing you would write to accomplish this task. All that needs done is to  add just a few lines of code to turn them into Stackstorm actions. Let's have a look!
+
+![](http://www.techworldwookie.com/blogpost/action.png)
```

Comment:
Could you instead get a code bloc, so folks could look at it like code instead of a picture? if not could you use https instead of http. I have tested on your site and it works with https. We get security warning when using http.